langchain/src/agents/executor.ts

```diff
@@ -77,9 +77,7 @@ export class AgentExecutor<
     agent_scratchpad?: string | BaseMessage[];
     stop?: string[];
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } = any,
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  RunOutput extends AgentAction | AgentFinish = any
```

Comment:
Will this not cause problems?

Reply:
Builds without issues, and all agent tests still pass so I don't think there's an issue?

Reply:
Let me look at it for a bit, this might weaken typing in a way we don't want